src/components/e-fullwidth.vue

```diff
@@ -71,29 +79,31 @@
       }
     },
     mounted() {
-      if (!this.resizeObserver) { // IE11 fallback.
+      if (!this.resizeObserver) {
```

Comment:
This scenario can be removed, since all modern browsers know the resizeObserver.